AdobeExtensionSample/src/main/java/com/foresee/adobeextensionsample/SampleApplication.java

```diff
@@ -0,0 +1,60 @@
+package com.foresee.adobeextensionsample;
+
+import android.app.Application;
+import android.util.Log;
+
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.Identity;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.UserProfile;
+import com.foresee.sdk.ForeSee;
+import com.foresee.sdk.ForeSeeAdobeExtension;
+
+public class SampleApplication extends Application {
```

Comment:
The iOS extension has a `ForeSeeAdobeExtensionDelegate` the client can implement to capture when the SDK has started. Is there something similar on Android? And, if so, should we demonstrate its usage?

Reply:
Yes. The `ForeSee.ForeSeeSDKConfigurationListener` has two callbacks which can be used in this scenario:
- onSDKReady()
- onFailedInitializingSDK()

Although due to a SDK issue I don't think the `onSDKReady()` callback is working properly...

Reply:
@awangvan will you create and link a new ticket for this issue? In the meantime, we should probably leave this out of the sample, if it's not working right 🤪

Reply:
Done